multiqc/modules/mirtop/mirtop.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from mirtop"""
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+
+from multiqc import config
+from multiqc.plots import bargraph, beeswarm
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='mirtop',
+        anchor='mirtop', target='Subread mirtop',
+        href='https://github.com/miRTop',
+        info="is a Command line tool to annotate miRNAs and isomiRs using a standard naming."
+        )
+
+        # Find and load any mirtop reports
+        self.mirtop_data = dict()
+        self.mirtop_keys = list()
+        for f in self.find_log_files('mirtop'):
+            self.parse_mirtop_report(f)
+        # Filter to strip out ignored sample names
+        self.mirtop_data = self.ignore_samples(self.mirtop_data)
+
+        if len(self.mirtop_data) == 0:
+            raise UserWarning
+
+        log.info("Found {} reports".format(len(self.mirtop_data)))
+
+        # Write parsed report data to a file
+        self.write_data_file(self.mirtop_data, 'multiqc_mirtop')
+        #log.info(self.mirtop_data)
+
+        # Create very basic summary table
+        self.mirtop_stats_table()
+
+        # Create comprehensive beeswarm plots of all stats 
+        self.add_section (
+             name = 'IsomiR Summary Statistics',
+             anchor = 'mirtop-stats',
+             description = "This module parses the summary data generated by <code>mirtop</code>. ",
```

Comment:
The description is parsed as markdown now, so you can just use backticks instead of HTML `<code>`.